gemfiles/rails_6.1.gemfile

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+# Specify your gem's dependencies in lograge.gemspec
+gemspec path: '..'
+
+group :test do
+  gem 'actionpack', '~> 6.1.0'
+  gem 'activerecord', '~> 6.1.0'
+  # logstash does not release any gems on rubygems, but they have two gemspecs within their repo.
+  # Using the tag is an attempt of having a stable version to test against where we can ensure that
+  # we test against the correct code.
+  gem 'logstash-event', git: 'https://github.com/elastic/logstash', tag: 'v1.5.6'
```

Comment:
Thanks for all your patience with this! It looks like everything but the JRuby specs are passing. From the looks of it, this version number changed between the other gemfiles:

```suggestion
  gem 'logstash-event', git: 'https://github.com/elastic/logstash', tag: 'v1.5.4'
```

With that change ☝️ I expect the build should pass (tested locally with JRuby 9.2 and 9.3). See also #354.

Reply:
should we not better use this tag on all  rails tests (`5.2`,`6.0`,`6.1` ...) ?

Reply:
We use logstash-event v1.5.4 currently:

https://github.com/roidrage/lograge/blob/c737a13779fef5c2b6050ce9356fd7acb86a8905/gemfiles/rails_5.2.gemfile#L15
https://github.com/roidrage/lograge/blob/c737a13779fef5c2b6050ce9356fd7acb86a8905/gemfiles/rails_6.0.gemfile#L14
https://github.com/roidrage/lograge/blob/c737a13779fef5c2b6050ce9356fd7acb86a8905/gemfiles/rails_edge.gemfile#L14

Bumping the version here would likely mean updating the formatter to be compatible with the API changes in logstash-event.

Reply:
Should not it be done like this ? I mean :
- the bump is only about patch level (pretty sure logstash api has no change)
- we should keep consistency and ensure that 1.5.4 and 1.5.6 work the same